src/osal/rt-kernel/osal.c

```diff
@@ -13,24 +13,19 @@
  * full license information.
  ********************************************************************/
 
-#include "osal.h"
-#include "options.h"
 #include "log.h"
+#include "options.h"
+#include "osal.h"
```

Comment:
Please keep the module header first in the list of includes (in all files).

Reply:
Yes, sorry.
Done.
Also cmake change to copy the new scripts.